client/src/selectors/schedules.js

```diff
@@ -27,13 +27,14 @@ export const isSelectorDisabled = (selectionSegments, status, startTime, endTime
   return disabled;
 };
 
-export const selectSchedules = (schedules, { startTime, endTime }) => {
+export const selectSchedules = (schedules, { userId, startTime, endTime }) => {
   const filteredItems = [];
   schedules.map(schedule =>
     schedule.timeSegments.map((timeSegment) => {
       if (
         (timeSegment.startTime === 0 || timeSegment.startTime >= startTime) &&
-        (timeSegment.endTime === 2147483647 || timeSegment.endTime <= endTime)
+        (timeSegment.endTime === 2147483647 || timeSegment.endTime <= endTime) &&
```

Comment:
constants.js distantFuture

Reply:
Yep this is really old code - I will tidy up